src/js/module.js

```diff
@@ -49,6 +49,19 @@ if (process.env.IOTJS_PATH) {
   moduledirs.push(process.env.IOTJS_PATH + '/iotjs_modules/');
 }
 
+function tryPath(modulePath, ext) {
+  var filepath;
+
+  if (filepath = iotjs_module_t.tryPath(modulePath)) {
+    return filepath;
+  }
+
+  if (filepath = iotjs_module_t.tryPath(modulePath + ext)) {
+    return filepath;
+  }
+
+  return filepath;
+}
 
```

Comment:
I think the code above is same to following:

```js
function tryPath(modulePath, ext) {
  return iotjs_module_t.tryPath(modulePath) ||
         iotjs_module_t.tryPath(modulePath + ext)
}
```

Reply:
@glistening Thanks, it looks better. I've changed it.